lib/fog/compute/ecloud/requests/ssh_key_delete.rb

```diff
@@ -0,0 +1,19 @@
+module Fog
+  module Compute
+    class Ecloud
+      class Real
+        basic_request :ssh_key_delete, 204, "DELETE"
+      end
+
+      class Mock
+        def ssh_key_delete(uri)
+          ssh_key_id = id_from_uri(uri)
+
+          ssh_key = self.data[:ssh_keys][ssh_key_id]
```

Comment:
Useless assignment to variable - `ssh_key`. Did you mean `ssh_key_id`?<br>Redundant `self` detected.